src/Common/src/System/Drawing/ColorTranslator.cs

```diff
@@ -260,6 +261,7 @@ public static Color FromHtml(string htmlColor)
                     InitializeHtmlSysColorTable();
                 }
 
+                Debug.Assert(s_htmlSysColorTable != null);
```

Comment:
Nit: I'd suggest moving this up to be inside the if block, after the initialize call.  Another case, though, where it'd also be fine to just use `!`